### PR TITLE
VIH-5454 change judge display name

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-5454
### Change description ###
Change judge display name in waiting room
**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts [hunk 1]
```diff
@@ -45,33 +49,66 @@ export class JudgeParticipantStatusListComponent implements OnInit {
...
+
+  private updateParticipant() {
+    const updateParticipantRequest = new UpdateParticipantRequest(
+      {
+        fullname: this.judge.name,
+        display_name: this.judge.display_name,
+        representee: this.judge.representee
+      });
+
+    this.videoWebService.updateParticipantBookingDetails(this.conference.id, this.judge.id, updateParticipantRequest)
```
Comment (RuwanikaWeerakkody) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts:109]: Hi Valentina, Can you confirm what you are trying to do here. The name "updateParticipantBookingDetails"  suggest that you are trying to update the booking database. But the code seems to call the Video API 
 let url_ = this.baseUrl + "/conferences/{conferenceId}/participants/{participantId}/participantDisplayName";

Can you confirm and rename based on above?

File: VideoWeb/VideoWeb/Controllers/ParticipantsController.cs [hunk 2]
```diff
@@ -139,6 +139,28 @@ public async Task<IActionResult> GetHeartbeatDataForParticipantAsync(Guid confer
...
+        public async Task<IActionResult> UpdateParticipantDisplayNameAsync(Guid conferenceId, Guid participantId, [FromBody] UpdateParticipantRequest participantRequest)
+        {
+            if (conferenceId == Guid.Empty || participantId == Guid.Empty)
+            {
+                return BadRequest($"Please provide a valid conference Id and participant Id");
+            }
+
+            try
+            {
+               await  _videoApiClient.UpdateParticipantDetailsAsync(conferenceId, participantId, participantRequest);
```
Comment (RuwanikaWeerakkody) [inline VideoWeb/VideoWeb/Controllers/ParticipantsController.cs:155]: Did you check if the correct display name gets passed to the live hearing after updating the display name in the video DB? Please check. Thanks.

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts (hunk 1 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.ts:109]: I renamed it , removed 'booking', it 's call video api endpoint to update display name in video database.

File: VideoWeb/VideoWeb/Controllers/ParticipantsController.cs (hunk 2 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/Controllers/ParticipantsController.cs:155]: we pass to live hearing    ParticipantResponse wich is mapped from conference.participant and booking participant, so display name will be mapped from video database =>
         var response = new ParticipantResponse
            {
                Id = participant.Id,
                FirstName = bookingParticipant?.First_name,
                LastName = bookingParticipant?.Last_name,
                Name = participant.Name,
                Status = status,
                Role = role,
                Username = participant.Username,
                DisplayName = participant.Display_name,
                CaseTypeGroup = participant.Case_type_group,
                Representee = participant.Representee,
                ContactEmail = bookingParticipant?.Contact_email,
                ContactTelephone = bookingParticipant?.Telephone_number
            };

**Pull request merged**